test/functional/basic/gesture-e2e-specs.js

```diff
@@ -159,11 +159,13 @@ describe('XCUITestDriver - gestures', function () {
       await driver.execute('mobile: scroll', {direction: 'down'});
       await driver.elementByAccessibilityId('Steppers').click();
 
+      await B.delay(1000);
```

Comment:
Can be a `retryInterval` for `await driver.elementByAccessibilityId('Increment');`?
(L168 as well)

Reply:
I tried with that and the test failed.